docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -33,7 +27,7 @@ The names of the following entities must be unique on the project level:
 
 ## Transfer name is not unique
 
-Transfer names must be unique. Check the examples below to learn how to fix the error.
+Transfer names must be unique for the new transfers on the project level. Below you can find the code examples that show how to solve the described problem.
```

Comment:
If I rename an existing transfer, will the evaluator parse it?

Reply:
We can't rename core transfer on the project level, only create a new one

Reply:
Maybe Andriis concern relates to the word 'new'. Probably Andrii means that it can be understood as 'new from the point when I start to use the evaluator'. But of course, it relates to all added transfers to the project.